CHANGELOG.md

```diff
@@ -3,6 +3,7 @@ All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [2.1.1] - Unreleased
+- Add loaded function to modules, providing an async callback.
```

Comment:
@roramirez It will be 2.1.1, 2.2.0 will be introduced when there are breaking changes (just like 2.1.0).